custom_components/tuya_local/devices/unknow_2way_energymeter.yaml

```diff
@@ -0,0 +1,452 @@
+# Can´t put some DPs to work (calibration and alarms)
+name: Bidirectional energy meter
+products:
+  - id: 8bjnabt4gh8tu4vr
+    name: Smart meters
+primary_entity:
+  entity: sensor
+  name: Forward energy total
```

Comment:
I don't think total is needed in the names.

And for this specific sensor, probably the default name is appropriate, so name can be removed completely.